nuklear.h

```diff
@@ -17230,7 +17230,7 @@ nk_font_bake_convert(void *out_memory, int img_width, int img_height,
     const void *in_memory)
 {
     int n = 0;
-    nk_rune *dst;
+    nk_byte *dst;
```

Comment:
Thanks a lot for the update. Would you mind making this change in the src destroy? We use paq.sh to combine the files, so wouldn't want to lose your changes.

Reply:
Thanks, I will do the needed changes today or tomorrow and I'll ping you back.

Reply:
I tested the changes on a little endian machine, the conversion still works.

I added the changes to the src directory as well; should I remove them from `nuklear.h`, or is it OK like it is now?